domains/DataCollection/Forms/IdentificationForm/index.js

```diff
@@ -57,8 +57,6 @@ const IdentificationForm = ({
 
             formObject.dob = `${values.Month || '00'}/${values.Day || '00'}/${values.Year || '0000'}`;
 
-            formObject.searchIndex = `${values.fname || ''} ${values.lname || ''}`;
```

Comment:
@jamccomb92 so we're going to push an array instead, a lot of the processing we're doing is because we're not pushing an array of the desired values from the frontend. I'll handle that. 

thanks for the catch!